src/main/kotlin/org/wycliffeassociates/otter/jvm/app/theme/AppColors.kt

```diff
@@ -8,6 +8,9 @@ abstract class AppColors {
     val appRed: Color = c("#CC4141")
     val appBlue: Color = c("#0094F0")
     val appGreen: Color = c("#58BD2F")
+    val appOrange: Color = c("F8A317")
+    val appLightGradientRed: Color = c("#E56060")
```

Comment:
not sure what I think of using the word gradient here for a single color

Reply:
Yeah, I just didn't think we'd be using it for anything else but I should probably name it something else. It's not exactly a light red either though lol